crypto.go

```diff
@@ -0,0 +1,50 @@
+package srtp
+
+import (
+	"crypto/cipher"
+)
+
+// xorBytes computes the exclusive-or of src1 and src2 and stores it in dst.
+// It returns the number of bytes written.
+func xorBytes(dst, src1, src2 []byte) int {
```

Comment:
Could you add test cases to ensure behavior against different combination of buffer length?

Reply:
Done.